docs/docs/api/multiplayer.md

```diff
@@ -96,23 +96,53 @@ Rune.initLogic({
 })
 ```
 
-## `Rune.gameOver()`
+## `Rune.gameOver(options)`
 
-When the game has ended (with a winner or draw), the action handler should call `Rune.gameOver()`. Your game doesn't need to show a "game over" screen. Rune overlays a standardized high score interface to the user.
+When the game has ended, the action handler should call `Rune.gameOver`. Your
+game doesn't need to show a "game over" screen. Rune overlays a standardized
+game over popup to the user. See more in the
+[Game Over](../multiplayer/game-over.md) guide.
 
 ```js
 // logic.js
 Rune.initLogic({
   actions: {
-    myAction: (payload, { game, playerId }) => {
-      if (isVictoryOrDraw(game)) {
-        Rune.gameOver()
+    myAction: (payload, { game }) => {
+      if (isGameOver(game)) {
+        const winner = getWinner(game)
+        const loser = getLoser(game)
+
+        Rune.gameOver({
+          players: {
+            [winner.playerId]: "WON",
+            [loser.playerId]: "LOST",
+          },
+          delayPopUp: true,
+        })
       }
     },
   },
 })
 ```
 
+### `players: Record<string, "WON" | "LOST" | number>`
+
+`players` is an object with player IDs as keys and the game result as values.
+The game result for each player can be either `WON`/`LOST` or an integer player
+score. Mixing `WON`/`LOST` and scores at the same time is not allowed. All
+players present in the game at the moment the game ends must be mentioned in
+the `players` object.
+
+### `delayPopUp?: boolean | undefined`
+
+Optional. Set to `true` if you want to instruct Rune to delay showing of the
+game over popup until you call `Rune.showGameOverPopUp()`.
+
+## `Rune.showGameOverPopUp()`
+
+If you set `delayPopUp` to `true` in `Rune.gameOver()`, you should call this
+function in your **client** code to show the game over popup.
```

Comment:
We usually just refer to `client.js`. Maybe mention motivation on why one might want to do this.

Reply:
I think we agreed some time ago that client.js is just a suggestion since the client code can really be anything. I think it's ok to use `client.js` when naming a hypothetical example file, but I'm not sure about just referring to it in the text

Reply:
Changed https://github.com/rune/rune-games-sdk/pull/128/commits/10b8f0991598b56f4348f617122f753ffeeded4c